.github/workflows/release.yml

```diff
@@ -0,0 +1,107 @@
+name: Release
+
+permissions:
+  contents: write
+  pull-requests: write
+
+env:
+  PANDOC_VERSION: 3.1.1.0
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - release
+
+jobs:
+  release-pr:
+    # guard against running on forks
+    if: github.repository == 'TTT-2/TTT2'
```

Comment:
Is there no "this" repository? Do you need to hardcode the name?

Reply:
"this" repository would mean the forked repo on a fork, which is exactly what i want to prevent. (besides i don't think that something like that even exists)